vhdbuilder/packer/test/linux-vhd-content-test.sh

```diff
@@ -962,6 +962,36 @@ testNBCParserBinary () {
 
 }
 
+testWasmRuntimesInstalled () {
+  local test="testWasmRuntimesInstalled"
+  local wasm_runtimes_path="/usr/local/bin"
+  local spin_runtime_versions="v0.3.0 v0.5.1"
```

Comment:
We should add v0.8.0 v0.15.1 to cover our bases on all of the versions we are expecting